pkg/dashboard/ui/package.json

```diff
@@ -85,6 +85,7 @@
     "protractor-jasmine2-screenshot-reporter": "^0.2.0"
   },
   "scripts": {
-    "run-dev": "gulp --dev"
+    "run-dev": "gulp --dev",
+    "run-watch": "gulp watch"
```

Comment:
What does this `watch` script do?

Reply:
reload html without the need to recompile ui files